### PR TITLE
German support proposal

### PR DESCRIPTION
**Pull request by rtxm**
Follow-up from #39

### CODE REVIEW

File: text_to_num/parsers.py [hunk 1]
```diff
@@ -181,6 +179,297 @@ def push(self, word: str, look_ahead: Optional[str] = None) -> bool:
...
+    digits by group of three to more easily speak them out.
+    And indeed, the language uses powers of 1000 to structure big numbers.
+
+    Public API:
+
+        - ``self.parse(word)``
+        - ``self.value: int``
+    """
+
+    def parse(self, word: str) -> bool:
```
Comment (rtxm) [inline text_to_num/parsers.py:197]: The signature says `parse` returns a `bool` but the function body has no return statement.

Comment (rtxm) [reply text_to_num/parsers.py:197]: The function is too long/complex.

**Comment by rtxm** [comment]
Please apply mypy and fix reported errors.

**Comment by fquirin** [comment]
I've merged the current master into the German branch and tried to fix the most important issues: https://github.com/allo-media/text2num/pull/55

Overall I think it's not optimized well yet, but I think my changes will make it more easy to continue :-)

**Comment by rtxm** [comment]
I'm quite busy for the moment, any volunteer to complete the README and documentation to add German examples?

**Comment by fquirin** [comment]
@rtxm I've updated the README and tweaked the German code a little bit: https://github.com/allo-media/text2num/pull/56

I'm not sure though how to update the "documentation" :sweat_smile:

**Pull request merged**